roles/httpd.lua

```diff
@@ -75,7 +76,22 @@ end
 -- parse_params returns table with set options from config to pass
 -- it into new() function.
 local function parse_params(node)
+    local log_requests = node.log_requests
+    if type(log_requests) == "string" then
+        local level = log_requests:lower()
+        if level == "info" then
+            log_requests = log.info
+        elseif level == "verbose" then
+            log_requests = log.verbose
+        elseif level == "debug" then
```

Comment:
Let's make all levels compatible with Tarantool: https://www.tarantool.io/en/doc/latest/reference/configuration/#confval-log_level

Reply:
Does it make sense to log regular requests at levels above `info` like `warn`, `crit` or `error`? Those levels are typically reserved for exceptional or error situations, not for normal request logging.